ci/parallel_sync/sync_to_tip.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+export LC_ALL=C
+set -Eeuo pipefail
+
+setup_vars() {
+  # Binaries
+  DEFID_BIN=${DEFID_BIN:-"./defid"}
+  DEFI_CLI_BIN=${DEFI_CLI_BIN:-"./defi-cli"}
+
+  # Files and directories
+  DATADIR=${DATADIR:-".defi"}
+  DEBUG_FILE="$DATADIR/debug.log"
+  CONF_FILE="$DATADIR/defi.conf"
+  TMP_LOG=debug-tmp-$STOP_BLOCK.log
+  BASE_PATH=https://storage.googleapis.com
+  BUCKET=team-drop
+  REF_LOG_DIR=master-logs-full
+  REF_LOG=debug-$STOP_BLOCK.log
+  REF_LOG_PATH=$BASE_PATH/$BUCKET/$REF_LOG_DIR/$REF_LOG
+
+  # Commands
+  DEFID_CMD="$DEFID_BIN -datadir=$DATADIR -daemon -debug=accountchange"
+  DEFI_CLI_CMD="$DEFI_CLI_BIN -datadir=$DATADIR"
+  ACCOUNT_BALANCES_CMD="$DEFI_CLI_CMD logaccountbalances"
+  LIST_ANCHORS_CMD="$DEFI_CLI_CMD spv_listanchors"
+  GREP="grep"
+
+  BLOCK=0
+  ATTEMPTS=0
+  START_BLOCK=${START_BLOCK:-0}
+  STOP_BLOCK=${STOP_BLOCK:-0}
+  MAX_ATTEMPTS=10
+  MAX_NODE_RESTARTS=5
+  NODE_RESTARTS=0
+}
+
+echo "======== Sync Test Info ==========
+- Block range: ${START_BLOCK} - ${STOP_BLOCK}
+- Base snapshot: https://gcr.io/br-blockchains-dev/datadir-${START_BLOCK}
```

Comment:
Can we get a direct link for this? I assume this leads to a dir.

Reply:
its the link to snapshot(tar) file